package.json

```diff
@@ -48,6 +48,7 @@
   "devDependencies": {
     "chai": "^3.4.1",
     "chai-as-promised": "^5.1.0",
+    "gulp-webpack": "^1.5.0",
```

Comment:
Needs to be in the other dep section as we run the build script on Heroku and Heroku does a `npm i --production`